tests/spec/core/xref-spec.js

```diff
@@ -4,14 +4,15 @@ describe("Core — xref", () => {
 
   let cache;
   beforeAll(async () => {
-    const { openDb } = await new Promise(resolve =>
-      require(["deps/idb"], resolve)
-    );
+    await new Promise(resolve => require(["deps/idb"], resolve));
+    const { openDB } = window.idb;
```

Comment:
I want to use some sort of shim here, so that I don't need to reference the window object. Just couldn't find the right documentation.

Reply:
you are only doing this in the tests, right?

Reply:
yes. Only in tests. I tried adding the shim to ```test-main.js```. doesn't help